GOVERNANCE.md

```diff
@@ -69,6 +99,49 @@ The Technical Committee meets regularly and the meetings are intended to be open
+  * After the commencement of the nomination phase, eligible candidates who wish to run for a seat in Technical Committee announce their nomination by sending an email to [Eiffel Community Maillist](https://groups.google.com/g/eiffel-community). (self-nomination) If community members want to nominate someone else to Technical Committee, they can do so by sending nomination mail to [Eiffel Community Maillist](https://groups.google.com/g/eiffel-community) but this must then be accepted by the nominated community member by responding to their nomination mail in order for their candidacy to be valid. Only maintainers can self-nominate or nominate other maintainers.
+  * Upon the completion of the nomination phase, Election Officers collate the nominations and announce final list of candidates on [Eiffel Community Maillist](https://groups.google.com/g/eiffel-community).
+  * Announcement of candidates ends the nomination phase.
+* **Election Phase**:
+  * Eiffel Community uses Condorcet Internet Voting Service System ([CIVS](https://civs.cs.cornell.edu/)) to hold elections. The instructions to prepare the election poll is available [here](https://wiki.opendaylight.org/display/ODL/CIVS+Voting+System+Information).
+  * Election Officers prepare the election poll on CIVS and enter candidates to ballot.
+  * Election Officers announce the start of election phase on [Eiffel Community Maillist](https://groups.google.com/g/eiffel-community). Election period is limited to **one week**.
+  * After the announcement, Election Officers send mails to community members who are eligible to vote in the election.
+  * Upon the completion of the election phase, Election Officers announce the result of the election on [Eiffel Community Maillist](https://groups.google.com/g/eiffel-community).
+
+Elected members of the Technical Committee assume their seat during the first Technical Committee Meeting following the completion of the election phase.
+
+#### Exceptions
+
+There could be cases where Election Officers may need to follow different processes to ensure the election process can be completed.
+
+One such case is the number of candidates who run for the elections.
+If the number of candidates is less than the number of seats in Technical Committee, Election Officers are authorized to extend the nominated period for one more week. If the number of candidates is still less than the number of seats in Technical Committee, Election Officers are authorized to end the nomination and election process so all the candidates assume their seats without election. The remaining seats are left vacant. The new Technical Committee is responsible to determine if, when, and how to fill the vacant seats.
```

Comment:
Should we have a reasoning about what happens if the election officers conclude that there is not enough interest in a TC to be able to run it?

Reply:
Good point.

In this case, we need to define the criteria to conclude that there is not enough interest in a TC. Something like "If the no of candidates is less than half of the total seats in TC" etc. 

If we could do that, we can then say, "Election Officers are authorized to halt/postpone the elections to consult the existing TC for the way forward" or something.

The rest is then up to TC - do some outreach/convincing, reduce the no of seats, etc.